doc/api/crypto.md

```diff
@@ -944,6 +944,15 @@ The `output_format` can specify one of `'latin1'`, `'hex'` or `'base64'`. If
 `output_format` is provided a string is returned; otherwise a [`Buffer`][] is
 returned.
 
+The optional `options` argument is an object which specifies additional
+cryptographic parameters. Currently, the following options are supported:
+
+* `padding` : {String} - RSA padding, either `'pkcs1'` for RSASSA-PKCS1-v1_5
```

Comment:
@tniessen Thanks for pointing that out – we’re moving to consistenly using lowercase in https://github.com/nodejs/node/pull/11697, so it would be good if this PR was in line with that. I’ll comment that I’d prefer to have the space before the colon dropped there, too.